data/nav.yml

```diff
@@ -11,8 +11,8 @@ nav_items:
     link: "/events/"
   - title: "Childcare"
     link: "/childcare/"
-  - title: "Scholarships"
-    link: "/scholarships/"
+  # - title: "Scholarships"
+  #   link: "/scholarships/"
```

Comment:
Just wondering, if it's not in the nav anymore, is it still possible to get to this page without manually typing in the slug?

Reply:
Is it linked in the place where we talk about diversity in `/about`?

I was thinking maybe we could have like a second level nav in the footer or something